tests/test_edge_detection.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import os
+import pytest
+
+import numpy as np
+import pandas as pd
+
+from pymagine import edge_detection as ed
+
+fname = os.path.join(os.path.dirname(__file__), '../tests/imgs/coronado_beach.jpeg')
+bad_ftype = 'imgs/coronado_beach.csv'
+url_fname = 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c3/Python-logo-notext.svg'
+
+def test_inputs():
```

Comment:
add docstring for test_inputs
```suggestion
def test_inputs():
"""
Runs tests to ensure edge_detection function is working properly.
"""
```